pkg/volume/local/local.go

```diff
@@ -207,6 +213,17 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 	if !notMnt {
 		return nil
 	}
+	if fsGroup != nil {
+		fsGroupSame, fsGroupOld, err := isSamefsGroup(m.globalPath, fsGroup)
```

Comment:
I'm mainly concerned that this is not a requirement that other volume plugins require.  The other plugins are also prone to this same issue, but not as often as local volumes.